app/models/commit_monitor_repo.rb

```diff
@@ -45,9 +55,8 @@ def with_github_service
 
   def with_travis_service
     raise "no block given" unless block_given?
-    client = Travis::Client.new
-    client.github_auth(Settings.github_credentials.password) # Assumes password is holding a token
-    repo = client.repo(fq_name)
-    yield repo
+
+    Travis.github_auth(Settings.github_credentials.password)
+    yield Travis::Repository.find(fq_name)
```

Comment:
Simplified this.  Travis::Repository.find is what we are returning anyway so call it directly instead of separately instantiating the client.  Note, on the repo, you can still get to the initialized client.